bridge-api/paths/compoundactivitydefinitions/v3_compoundactivitydefinitions.yml

```diff
@@ -0,0 +1,40 @@
+get:
+    operationId: getAllCompoundActivityDefinitionsInStudy
```

Comment:
So far we have reserved "InStudy" for the worker calls that require specifying a specific study... if this is a developer API it is presumed to be study-scoped, so I would call this "getAllCompoundActivityDefinitions" in case we need to add a worker version later.